### ISSUE TITLE
Error in macro 'completion'

### ISSUE THREAD

**Issue by rubenvereecken** [porras/tlcr#1]
I'm not a Crystal guy, so I'll just drop this here.

```
Error in macro 'completion' tlcr/libs/completion/completion/macros.cr:1, line 4:

  1.   __temp_148 = Completion::Completer.new File.basename $0
  2.   __temp_148.set_fragments [:command]
  3.   comp = __temp_148
  4.   comp.reply(:command, client.index.available.map(&.name))
  5.   __temp_148.init

  comp.reply(:command, client.index.available.map(&.name))
       ^~~~~

wrong number of arguments for 'Completion::Completer#reply' (2 for 1)
Overloads are:
 - Completion::Completer#reply(results)
```

**Comment by porras** [comment]
Hi!

Thanks for reporting. I'm trying to reproduce it, could you please provide a bit more information on the environment? Specifically:
- Crystal version (`crystal -v`)
- f/completion version (it should be ok because it's locked in `shard.lock` but it's worth checking that running `shards` didn't change that file)
- OS, etc.

Thanks!

**Comment by porras** [comment]
Hi!

I reproduced this when building from scratch. The dependency on `f/completion` was not correctly locked so a new version of it broke `tlcr`. It's fixed in 19db663. Thanks for reporting!
